src/js/jsx/DocumentHeader.jsx

```diff
@@ -167,6 +178,15 @@ define(function (require, exports, module) {
                     <div className="document-header" ref="tabContainer">
                             {documentTabs}
                     </div>
+                    <div className="generator-header">
+                        <SplitButtonList>
+                            <SplitButtonItem
```

Comment:
I won't make a federal case out of this, but I don't see why this should be a split button as opposed to a regular button.  I could be wrong about this, but I think SplitButton is sort of a historical relic and should be used sparingly, and only for sets of buttons that are functionally and perhaps interactively joined somehow.